pkg/kubectl/cmd/cp/cp.go

```diff
@@ -275,6 +284,24 @@ func (o *CopyOptions) copyToPod(src, dest fileSpec, options *exec.ExecOptions) e
 	return o.execute(options)
 }
 
+func (o *CopyOptions) checkTarExists(pod fileSpec) error {
+	options := &exec.ExecOptions{
+		StreamOptions: exec.StreamOptions{
+			IOStreams: genericclioptions.IOStreams{
+				Out:    bytes.NewBuffer([]byte{}),
+				ErrOut: bytes.NewBuffer([]byte{}),
+			},
+			Namespace: pod.PodNamespace,
+			PodName:   pod.PodName,
+		},
+
+		Command:  []string{"which", "tar"},
```

Comment:
This will make both which and tar required inside the container. Today we only require tar. Have you thought about running `tar --version` or `tar --help` to avoid having to use `which`?

Reply:
Great suggestion. Implemented!